src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -357,6 +365,12 @@
     text: az aks update -g MyResourceGroup -n MyManagedCluster --load-balancer-outbound-ip-prefixes <ip-prefix-resource-id-1,ip-prefix-resource-id-2>
   - name: Attach AKS cluster to ACR by name "acrName"
     text: az aks update -g MyResourceGroup -n MyManagedCluster --attach-acr acrName
+  - name: Update a kubernetes cluster with authorized apiserver ip ranges.
+    text: az aks update -g MyResourceGroup -n MyManagedCluster --api-server-authorized-ip-ranges 193.168.1.0/24,194.168.1.0/24
+  - name: Disable authorized apiserver ip ranges feature for a kubernetes cluster.
+    text: az aks update -g MyResourceGroup -n MyManagedCluster --api-server-authorized-ip-ranges ""
+  - name: Allow all traffic to an apiserver in a kubernetes cluster but don't disable authorized ip ranges feature to allow for future use.
```

Comment:
this message feels obscure. I'd prefer "disallow all external traffic to the apiserver", thoughts?

Reply:
Yup, your right - this wording is off. Updating the wording